src/SortingAlgorithms.jl

```diff
@@ -560,4 +562,55 @@ function sort!(v::AbstractVector, lo::Int, hi::Int, ::TimSortAlg, o::Ordering)
     return v
 end
 
+
+## Intro sort
+
+function sort!(v::AbstractVector, lo::Int, hi::Int, a::IntroSortAlg, o::Ordering)
+    N = length(v)
+    if N > 1
```

Comment:
So this line should just be `hi-li > 1`, will see if that works and push an update commit here.

Any thoughts on optimizations for `index_of_median_of_three` ?